src/test-utils/dom/app-layout/index.ts

```diff
@@ -62,7 +62,9 @@ export default class AppLayoutWrapper extends ComponentWrapper {
   }
 
   findDrawersTriggers(): ElementWrapper<HTMLButtonElement>[] {
-    return this.findAllByClassName<HTMLButtonElement>(testutilStyles['drawers-trigger']);
+    return this.findAll<HTMLButtonElement>(
+      `.${testutilStyles['drawers-trigger']}:not([data-testid="awsui-app-layout-trigger-awsui-internal-tools"])`
```

Comment:
There is a constant for this value: https://github.com/cloudscape-design/components/blob/e1c06524f6ed70ce60edf02cf1d2b8295db39dfd/src/app-layout/utils/use-drawers.ts#L13

But we cannot use it here because of CJS/ESM compatibility issues. This is why a separate value